shop/src/main/java/ro/msg/learning/shop/repositories/CustomerRepository.java

```diff
@@ -0,0 +1,9 @@
+package ro.msg.learning.shop.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.entities.Customer;
+
+@Component
```

Comment:
@Component is a generic stereotype for any Spring-managed component. However, here we can use @Repository, which annotates classes at the persistence layer. Why? https://www.baeldung.com/spring-component-repository-service